Packs/GreyNoise/Integrations/GreyNoise/GreyNoise.yml

```diff
@@ -0,0 +1,519 @@
+category: Data Enrichment & Threat Intelligence
+commonfields:
+  id: GreyNoise
+  version: -1
+configuration:
+- display: API Key
+  name: apikey
+  required: true
+  type: 4
+- additionalinfo: Whether to allow connections without verifying SSL certificates
+    validity.
+  display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
```

Comment:
You create the parameter but do not use it at all in the py file
If you do not use it you can remove it.

Reply:
GreyNoise always works on secure communication so we have not used it anywhere, the reason for keeping it in YML file is because it is mandatory to keep both proxy and insecure fields as per "XSOAR Partner Design Document - Hello World".